universum/modules/structure_handler.py

```diff
@@ -138,7 +138,9 @@ def execute_one_step(self, configuration: Step,
         process = step_executor(configuration)
 
         background = configuration.background
-        process.start(is_background=background)
+        is_running = process.start(is_background=background)
```

Comment:
This way the encapsulation of the RunningStep class is broken.
It should be able to handle command absence on its own.

Reply:
Though just in case I need to point out that without breaking the encapsulation the background steps without command are still handled as actual background steps.